app/code/GorbanSv/CustomWidget/view/frontend/templates/widget/banner.phtml

```diff
@@ -0,0 +1,16 @@
+<?php
+/* @var $block \GorbanSv\CustomWidget\Block\Widget\Banner */
+?>
+
+<?php if ($block->getData('button_title')): ?>
+    <button class="button-cms-page">
+        <a href="<?php echo $block->getUrl('geekhub-cms') ?>"><?php echo $block->getData('button_title') ?></a>
+    </button>
+<?php endif; ?>
+
+<?php if ($block->getData('cms_block')): ?>
+    <?php echo $block->getLayout()
```

Comment:
Please move this procedure to your block \GorbanSv\CustomWidget\Block\Widget\Banner
Add the method getCmsBlockHtml() and perform these actions there